common/src/main/java/com/wynntils/core/consumers/overlays/Overlay.java

```diff
@@ -76,8 +110,43 @@ public void renderPreview(
         this.render(guiGraphics, bufferSource, deltaTracker, window);
     }
 
+    public void renderEnabledTemplateErrorMessage(GuiGraphics guiGraphics, MultiBufferSource bufferSource) {
```

Comment:
This should move to OverlayManager and be made private.

Reply:
But in that case at least make it package-private (that is, no visibility qualifier at all).

Reply:
Hm. Or maybe it need to be protected in that case.

Reply:
Actually, I think it should be made private with the new `renderOrErrorMessage`. If we ever want to override this we can loosen it up.